frontend/package.json

```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "dev": "vite",
-    "build": "tsc && vite build",
+    "build": "tsc && node --max_old_space_size=8192 ./node_modules/vite/bin/vite.js build",
     "watch": "tsc && vite build --watch",
```

Comment:
```suggestion
    "watch": "tsc && node --max_old_space_size=8192 ./node_modules/vite/bin/vite.js build --watch",
```

here too

Reply:
also we can remove this line 
https://github.com/microsoft/sample-app-aoai-chatGPT/pull/845/files#diff-014228303dff9a1af15f4bbd18401f906380129b10ae2a2c62f8b8be592ff88eR33

Reply:
Thanks, done!